test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -363,14 +362,18 @@ public void CreateDescriptor_BuildsDescriptorsWithInheritedProperties()
         {
             // Arrange
             var errorSink = new ErrorSink();
-            var intProperty = typeof(InheritedSingleAttributeTagHelper).GetProperty(
-                nameof(InheritedSingleAttributeTagHelper.IntAttribute));
+
+            // Also confirm isStringProperty is calculated correctly.
```

Comment:
Yes. I'm removing these tests since we aren't worrying about the scenario at all.